addon-api/content-script/Tab.js

```diff
@@ -823,4 +824,9 @@ export default class Tab extends Listenable {
   prompt(title, message, defaultValue, opts) {
     return modal.prompt(this, title, message, defaultValue, opts);
   }
+  isScratchAprilFools23() {
+    if (!this.redux.state) return true; // better safe than sorry
+    if (typeof this.redux.state?.scratchGui?.timeTravel === "object") return true;
+    return false;
+  }
```

Comment:
Here, it would have been more suitable to check for the session flag:
```js
addon.tab.redux.state.session.session.flags.everything_is_totally_normal
=== true
```
The code that shipped to users unintentionally disabled addons before April Fools because the timeTravel object is available, but not accessible through the UI

Reply:
This mistake costed a release